final release/format_data.py

```diff
@@ -1,200 +1,266 @@
-import numpy as np
+"""Functions for formatting, preprocessing, and augmentation."""
+
+import sys, os
 import pickle
-import matplotlib.pyplot as plt
+
+from astropy.coordinates import SkyCoord
+import astropy.io.ascii  as asc
+import astropy.units as u
 import h5py
+from keras.preprocessing.image import ImageDataGenerator
+from keras.utils import np_utils
+import numpy as np
+import matplotlib.pyplot as plt
 from skimage.transform import resize
-from astropy.io import ascii
-#import ask_first
 from sklearn.utils import shuffle
-from keras.utils import np_utils
-from astropy.coordinates import SkyCoord
-import astropy.units as u
-
-import sys, os
-
 
-def get_data(first_path, second_path, dim = 256, seed = 0, conf_matrix = True, flip_train_test = False):
-    # Loading data
-    data = h5py.File(first_path, 'r')
 
-    images = np.asarray(data['images'])
-    labels = np.asarray(data['labels'])
+# Defining rescaling function
```

Comment:
Drop this comment (I can already tell that `def rescale` defines a rescaling function!)

Reply:
ok